stock_picking_invoice_link/models/sale_order.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# © 2013-15 Agile Business Group sagl (<http://www.agilebg.com>)
+# © 2017 Jacques-Etienne Baudoux <je@bcim.be>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.multi
+    def invoice_line_create(self, invoice_id, qty):
+        self.mapped('procurement_ids') \
+            .mapped('move_ids') \
```

Comment:
Why add unnecessary line breaks if they fit in the same line. This line can be styled with no need of backslash break.